subsys/net/lib/lwm2m/lwm2m_message_handling.c

```diff
@@ -2915,26 +2915,6 @@ int do_composite_read_op_for_parsed_list(struct lwm2m_message *msg, uint16_t con
 }
 
 #if defined(CONFIG_LWM2M_SERVER_OBJECT_VERSION_1_1)
-static int do_send_reply_cb(const struct coap_packet *response, struct coap_reply *reply,
-			    const struct sockaddr *from)
-{
-	uint8_t code;
-
-	code = coap_header_get_code(response);
-	LOG_DBG("Send callback (code:%u.%u)", COAP_RESPONSE_CODE_CLASS(code),
-		COAP_RESPONSE_CODE_DETAIL(code));
-
-	if (code == COAP_RESPONSE_CODE_CHANGED) {
-		LOG_INF("Send done!");
-		return 0;
-	}
-
-	LOG_ERR("Failed with code %u.%u. Not Retrying.", COAP_RESPONSE_CODE_CLASS(code),
-		COAP_RESPONSE_CODE_DETAIL(code));
-
-	return 0;
-}
-
```

Comment:
Leave this callback as a default callback.

Reply:
Done